static/app/utils/membersAndTeams/useAssignedValues.tsx

```diff
@@ -0,0 +1,57 @@
+import {useMemo} from 'react';
+
+import {
+  ItemType,
+  type SearchGroup,
+} from 'sentry/components/deprecatedSmartSearchBar/types';
+import {escapeTagValue} from 'sentry/components/deprecatedSmartSearchBar/utils';
+import {IconStar, IconUser} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import MemberListStore from 'sentry/stores/memberListStore';
+import TeamStore from 'sentry/stores/teamStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {getUsername} from 'sentry/utils/membersAndTeams/userUtils';
+
+export default function useAssignedValues(): SearchGroup[] {
```

Comment:
nit: naming is a bit vague, would prefer if it referenced the search in some way like `useSearchAssignedValues` and had a comment explaining what it does